src/py/mat3ra/made/tools/build/__init__.py

```diff
@@ -0,0 +1,100 @@
+from typing import Dict, Union, List
+from pymatgen.analysis.interfaces.coherent_interfaces import CoherentInterfaceBuilder, ZSLGenerator
+from pymatgen.analysis.structure_analyzer import SpacegroupAnalyzer
+from pymatgen.core.structure import Structure
+import numpy as np
+
+from .interface import InterfaceDataHolder, StrainModes, patch_interface_with_mean_abs_strain
+from ..utils import translate_to_bottom_pymatgen_structure
+from ..convert import convert_material_args_kwargs_to_structure
+
+
+@convert_material_args_kwargs_to_structure
+def create_interfaces(substrate: Structure, layer: Structure, settings, **kwargs):
```

Comment:
1. Settings needs type
2. We can spell out the kwargs directly
3. Return value type can to to the declaration